static/src/javascripts/projects/common/modules/experiments/tests/contributions-membership-epic-cta-united-states.js

```diff
@@ -0,0 +1,325 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic-equal-buttons.html',
+    'text!common/views/contributions-epic.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/modules/experiments/embed',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEpicEqualButtons,
+             contributionsEpic,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             embed,
+             ajax,
+             commercialFeatures
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsMembershipEpicCtaUnitedStates';
+        this.start = '2016-11-07';
+        this.expiry = '2016-11-11';
+        this.author = 'Jonathan Rankin';
+        this.description = '1) Find optimal way to present contributions and membershuip asks in Epic component. 2) Test 3 different messages for the Epic';
+        this.showForSensitive = true;
+        this.audience = 1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Impressions to number of contributions/supporter signups';
+        this.audienceCriteria = 'All readers in the US reading about US politics OR the US election, as well as not on Brexit articles ';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'We learn the best way to present contributions and membership asks in Epic component, and we lean what the most effective of the 3 messages is';
```

Comment:
You're testing multiple variables in each of your variants here, right? How do you know that it's the message or the presentation that works best? It feels like it would be better to test *the same presentation* with different messages and *the same messages* with different presentations - as seperate A/B tests?